<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q::vtk::PlanarSeedWidget</class>
 <widget class="QWidget" name="Q::vtk::PlanarSeedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlanarSeedWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonDeleteAll">
     <property name="text">
      <string>Delete All</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxNavigation">
     <property name="text">
      <string>Navigation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonAddOne">
     <property name="text">
      <string>Add One</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonDeleteOne">
     <property name="text">
      <string>Delete One</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QListWidget" name="listWidgetSeeds"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
